### PR TITLE
Improve kubernetes_sd_config in config doc

### PR DESCRIPTION
**Pull request by MIBc**
Add some meta labels which have writen in code.

Signed-off-by: MIBc <lvjiawei@cmss.chinamobile.com>

### CODE REVIEW

File: docs/configuration/configuration.md [hunk 1]
```diff
@@ -719,6 +719,8 @@ Available meta labels:
 * `__meta_kubernetes_service_port_name`: Name of the service port for the target.
 * `__meta_kubernetes_service_port_number`: Number of the service port for the target.
 * `__meta_kubernetes_service_port_protocol`: Protocol of the service port for the target.
+* `__meta_kubernetes_service_cluster_ip`: The cluster ip of the service.
```
Comment (brian-brazil) [inline docs/configuration/configuration.md]: These are already there

Comment (MIBc) [reply docs/configuration/configuration.md]: Sorry, I just notice it. The website don't list them.

File: docs/configuration/configuration.md [hunk 2]
```diff
@@ -781,6 +781,7 @@ Available meta labels:
 * `__meta_kubernetes_ingress_scheme`: Protocol scheme of ingress, `https` if TLS
   config is set. Defaults to `http`.
 * `__meta_kubernetes_ingress_path`: Path from ingress spec. Defaults to `/`.
+* `__meta_kubernetes_ingress_host`: A domain name of network host which from ingress.spec.rules.
```
Comment (brian-brazil) [inline docs/configuration/configuration.md:784]: This description doesn't seem right, wouldn't this just be the host without any reference to k8 internals?

**Comment by brancz** [comment]
@MIBc This still seems useful, did you want to apply the change regarding the documentation comment?

**Comment by bwplotka** [comment]
Hello from the Bug Scrub!

Looks like there was no activity for the last 3y, let's close it, we can always reopen. Thanks!

@darshanime mentioned they can verify if all labels from Kubernetes SD are in our configuration.

**Pull request closed without merging**